cli/azd/resources/apphost/templates/resources.bicept

```diff
@@ -371,4 +413,9 @@ output SERVICE_BINDING_{{alphaSnakeUpper $name}}_ENDPOINT string = {{bicepName $
 {{range $name, $value := .CosmosDbAccounts -}}
 output SERVICE_BINDING_{{alphaSnakeUpper $name}}_NAME string = {{bicepName $name}}.name
 {{end -}}
+{{range $name, $value := .SqlServers -}}
+{{range $cname := $value.Databases -}}
+output SERVICE_BINDING_{{alphaSnakeUpper $cname}}_CONNECTION_STRING string = 'Server=tcp:{{bicepName $name}}.Data.FullyQualifiedDomainName,1433;Initial Catalog={{bicepName $cname}};Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;Authentication=\'Active Directory Default\';'
+{{end -}}
+{{end -}}
```

Comment:
remove the `{{range $cname := $value.Databases -}}`
We just need one output per SqlServer (not per db).  All dbs from a server would have the same output and one SqlServer with no dbs needs to have an output (right now, it would not have an output if there's not a db added)